styles/main.css

```diff
@@ -159,6 +163,9 @@ footer hr {
     text-align: right;
     margin-right: -0.5em;
     opacity: 0.5;
+}
+.back-to-top a {
+    position: fixed;
     bottom: 3.5em;
 }

```

Comment:
The HYPERLINK needs the fixed position.  You also forgot to specify the `position:fixed` property.